### ISSUE TITLE
Files not in folder 'output'

### ISSUE THREAD

**Issue by ErikHogenbirk** [XENON1T/processing#12]
I have produced some simulation data using the [README](https://github.com/XENON1T/processing/tree/master/montecarlo), which is very clear and works fine *almost* all the way for me :) In step 11, the organize script expects that all files are in a folder named ```output``` which for me aren't. I suppose that this should be explicit in step 10, i.e.
```/project/lgrandi/xenon1t/simulations/mc_v<MC_VERSION>/pax_v<PAX_VERSION>/<MC_FLAVOR>/<MC_CONFIG>/output```
In my case, since I already untarred 10000 files, the argument list was event too long for a simple ```mv``` so that I fixed this with ```for f in *; do mv -v $f output/; done```, just for future reference.
Do you agree on this? In that case I'll just update step 10 in the README.

**Comment by pdeperio** [comment]
I agree; this part hasn't been carefully designed yet.  Feel free to improve the instructions/methods.  FYI, I changed the script a bit to make soft links instead of moving.
